packages/backend/src/controllers/users/users.controller.ts

```diff
@@ -23,19 +30,25 @@ export class UsersController {
 
   @Post()
   @ApiOperation({
-    summary: 'create a new user resource',
+    summary: 'Create a new user resource',
     description: 'User must be linked to a firebase ID',
   })
-  @ApiCreatedResponse({
+  @ApiBadRequestResponse({
     description: 'User successfully created',
     type: UserResponseDto,
   })
+  @ApiBadRequestResponse({ description: 'no user name provided' })
```

Comment:
There are two `@ApiBadRequestResponse` annotations and no `@ApiCreatedResponse`. We should have one of each to represent the two response types